screens/screen_todo.py

```diff
@@ -0,0 +1,32 @@
+from core.page_object import *
+from selenium.webdriver.common.keys import Keys
+from core.todo import Todo
+from locators import CreateToDo, ExistingToDO
+
+
+class ScreenToDo:
+    def create_new_todo(self, to_do, driver):
```

Comment:
create global_objects.py and during framework instatiation, save driver there.
Then, import driver from global_objects to any module. Then you won't have to use it as an argument in so many places